Blockzilla/BrowserViewController.swift

```diff
@@ -368,7 +368,13 @@ class BrowserViewController: UIViewController {
 
         urlBar.snp.makeConstraints { make in
             urlBarTopConstraint = make.top.equalTo(mainContainerView.safeAreaLayoutGuide.snp.top).constraint
-            make.leading.trailing.bottom.equalTo(urlBarContainer)
+            if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
Ok, indeed, I should also check for the split screen case. I will test it and make the necessary changes. Thank you!

Reply:
Through the `traitCollection` property you can find the horizontal and vertical size class. That is probably best indication on how wide or tall the screen is without checking for specific device types or layouts.

Reply:
I updated the implementation to check also for trait Collection for split view. Some of the implementation was in place,  and I used it. The already implemented solution for checking orientation/device/split view in the project did not use the size classes because somehow the size classes do not give the regular dimensions accurately in all the cases.  Also, it took me some time because during the transition to split view, where more changes to add for the constraints to update, since with this new design the URL bar has a little more differences on iPad.
Here are some screenshots

![Simulator Screen Shot - iPad mini 4 - 2021-08-12 at 13 46 53](https://user-images.githubusercontent.com/46751540/129205857-dc3b5365-6d78-4b25-a47e-827510df1b87.png)
![Simulator Screen Shot - iPad mini 4 - 2021-08-12 at 13 45 14](https://user-images.githubusercontent.com/46751540/129205904-bc59b314-4642-4e65-903a-d7ba44d383df.png)
![Simulator Screen Shot - iPad mini 4 - 2021-08-12 at 13 48 52](https://user-images.githubusercontent.com/46751540/129205946-39cd49ed-ebb5-4220-b051-9a74c68cc6e6.png)
![Simulator Screen Shot - iPad mini 4 - 2021-08-12 at 16 34 45](https://user-images.githubusercontent.com/46751540/129206114-72d5ce45-879b-44f2-a6e0-58dc763d1b21.png)

Reply:
Thank you for investigating - this looks great.